admin/orders.php

```diff
@@ -1251,16 +1251,19 @@ function couponpopupWindow(url) {
                 <td class="dataTableContent text-right"><?php echo strip_tags($currencies->format($orders->fields['order_total'], true, $orders->fields['currency'], $orders->fields['currency_value'])); ?></td>
                 <td class="dataTableContent text-right dataTableButtonCell">
                     <?php
-                    $sql = "SELECT op.products_quantity AS qty, op.products_name AS name, op.products_model AS model, opa.products_options AS product_option, opa.products_options_values AS product_value 
+                    $sql = "SELECT op.orders_products_id, op.products_quantity AS qty, op.products_name AS name, op.products_model AS model
                             FROM " . TABLE_ORDERS_PRODUCTS . " op 
-                            LEFT OUTER JOIN " . TABLE_ORDERS_PRODUCTS_ATTRIBUTES . " opa ON op.orders_products_id=opa.orders_products_id
-                            WHERE op.orders_id = " . (int)$orders->fields['orders_id'];
+                            WHERE op.orders_id = " . (int)$orders->fields['orders_id']; 
                     $orderProducts = $db->Execute($sql, false, true, 1800);
                     $product_details = '';
                     foreach($orderProducts as $product) {
                         $product_details .= $product['qty'] . ' x ' . $product['name'] . ' (' . $product['model'] . ')' . "\n";
-                        if (!empty($product['product_option'])) {
-                            $product_details .= '&nbsp;&nbsp;- ' . $product['product_option'] . ': ' . zen_output_string_protected($product['product_value']) . "\n";
+                        $sql2 = $db->Execute("SELECT products_options, products_options_values 
+                            FROM " .  TABLE_ORDERS_PRODUCTS_ATTRIBUTES . " 
+                            WHERE orders_products_id = " . (int)$product['orders_products_id'], false, true, 1800); 
```

Comment:
Shouldn't there be an ORDER BY clause here, e.g. `ORDER BY orders_products_attributes_id ASC`?